apis/apis.json

```diff
@@ -509,19 +511,32 @@
     }
   },
 
+  {
+    "id": "Google.Cloud.Spanner.ResourceNames.V1",
+    "type": "other",
+    "targetFrameworks": "netstandard1.5;net45",
+    "version": "2.0.0-beta00",
```

Comment:
Will this version be pinned to the others, or just the major version?

Reply:
I'm not quite sure what you mean by this, but I suspect we'll release a 2.0.0 of this (only 2.0.0 rather than 1.0.0 to avoid confusion) and then it may stay there forever, while all the other packages increase in version number as new functionality is added.

If we change codegen for resource names - e.g. to build a DatabaseName from a ProjectName and a string - we could update to 2.1.0 at that point.

Does that make sense and answer the question?

Reply:
Yes, basically, it was an observation that this started at 2... instead of 1... and that all the packages now use 2.0.0-beta00. I was wondering if the goal was to keep them all in sync (which I guess doesn't make sense since that all spanner packages would need to be updated/republished together) or if that was just a coincidence because this one happens to start at 2 to avoid confusion. I assume if the spanner packages go to 3.x, this one would too as well, to avoid confusion, right? In that sense, its major version would be pinned to the others.

Reply:
This one is called 2 just for convenience. The other ones going up to 2 is because this change will be a breaking one - we'll be removing the classes from the other projects.

We were planning to release Spanner.Data as 2.0 without updating the API libraries, but this changed that. I can definitely imagine Spanner.Data going to 3.0 without needing to change anything else. I don't mind starting at a non-1 value, but I think I'd *prefer* to avoid bumping major version numbers when there aren't breaking changes. All is up for negotiation though.